recipe/bld.bat

```diff
@@ -0,0 +1,11 @@
+setlocal EnableDelayedExpansion
+
+:: Copy the [de]activate scripts to %LIBRARY_PREFIX%\etc\conda\[de]activate.d.
+:: This will allow them to be run on environment activation.
+FOR %%F IN (activate deactivate) DO (
+    IF NOT EXIST %LIBRARY_PREFIX%\etc\conda\%%F.d MKDIR %LIBRARY_PREFIX%\etc\conda\%%F.d
+    COPY %RECIPE_DIR%/%%F.bat %LIBRARY_PREFIX%\etc\conda\%%F.d\keras_%%F.bat
```

Comment:
@msarahan, ah got it. Seems like it tries to call the keras activate script, but this fails because my CMD syntax is odd. I'm now suspecting that the syntax `set MY_KEY='secret-key-value'` in the [docs](https://conda.io/docs/using/envs.html#windows) is not valid because of the single quotes. Checking that.